packages/suite-web-landing/package.json

```diff
@@ -7,28 +7,29 @@
         "type-check:watch": "yarn type-check -- --watch",
         "lint": "eslint . --ext .tsx,.ts",
         "copy-files": "yarn workspace @trezor/suite-data copy-static-files",
-        "dev": "yarn workspace @trezor/suite-data copy-static-files && next",
-        "build:app": "next build && next export -o build",
-        "build": "run-s copy-files build:app"
+        "dev": "rimraf ./build && yarn workspace @trezor/suite-build run dev:landing",
+        "build": "rimraf ./build && yarn workspace @trezor/suite-build run build:landing"
     },
     "dependencies": {
         "@trezor/components": "1.0.0",
-        "next": "^10.0.5",
-        "polished": "^4.0.5",
-        "react": "17.0.1",
-        "react-awesome-reveal": "^3.5.2",
-        "react-dom": "17.0.1",
-        "react-intl": "^5.10.19"
+        "@trezor/suite": "21.9.0",
```

Comment:
@trezor/suite, redux and related should not be dependency of landing-page

Reply:
68c7a59